python/tank/deploy/tank_commands/setup_project_params.py

```diff
@@ -285,86 +285,86 @@ def get_configuration_shotgun_info(self):
+                                  "project",
                                   "project.Project.tank_name"])
-        
+
         return data
-    
+
     def get_configuration_readme(self):
         """
         Returns the contents of a configuration README file, if such a file
-        exists. 
-        
+        exists.
+
         :returns: list of strings or empty list if readme doesn't exist
         """
         if self._config_template is None:
             raise TankError("Please specify a configuration template!")
-        
+
         return self._config_template.get_readme_content()
-        
-    
+
+
     def get_required_storages(self):
         """
         Returns a list of storage names which are required for this project.
-        
+
         :returns: list of strings
         """
         if self._config_template is None:
             raise TankError("Please specify a configuration template!")
-        
+
         return self._storage_data.keys()
 
     def get_storage_description(self, storage_name):
         """
         Returns the description of a storage required by a configuration
-        
+
         :param storage_name: Storage name
         :returns: Storage description string, None if not defined
         """
         if self._config_template is None:
             raise TankError("Please specify a configuration template!")
-        
+
         if storage_name not in self._storage_data:
-            raise TankError("Configuration template does not contain a storage with name '%s'!" % storage_name) 
-        
+            raise TankError("Configuration template does not contain a storage with name '%s'!" % storage_name)
+
         return self._storage_data.get(storage_name).get("description")
-        
+
     def get_storage_path(self, storage_name, platform):
-        """    
+        """
         Returns the storage root path given a platform and a storage, as defined in Shotgun
         Note that this path has not been cleaned up, and may for example contain slashes at the end.
-        
+
         :param storage_name: Storage name
-        :param platform: operating system, sys.platform syntax 
+        :param platform: operating system, sys.platform syntax
         :returns: path
         """
         if self._config_template is None:
             raise TankError("Please specify a configuration template!")
-        
+
         if storage_name not in self._storage_data:
-            raise TankError("Configuration template does not contain a storage with name '%s'!" % storage_name) 
-        
+            raise TankError("Configuration template does not contain a storage with name '%s'!" % storage_name)
+
         return self._storage_data.get(storage_name).get(platform)
 
     def create_configuration(self, target_path):
         """
         Sets up the associated template configuration. Copies files.
-        
-        :param target_path: Location where the config should be set up. 
+
```

Comment:
See, that's what I meant yesterday by let's run a script and be done with it. :(